ui/src/components/Form/form-fields/TextField/index.js

```diff
@@ -11,7 +11,7 @@ const TextField = ({className, small=false, label, disabled, tooltipText, type="
     return (
         <div className={classnames("form-field-wrapper", "text-field", {small}, className)}>
             {!!label && <FieldLabel tooltipId={`form-tooltip-${field.name}`} tooltipText={tooltipText}>{label}</FieldLabel>}
-            <input type={type} {...field} className="form-field" disabled={disabled} />
+            <input type={type} {...field} className="form-field" disabled={disabled} {...(props.min && { "min": props.min })} />
```

Comment:
This will block the user from entering negative values using the field arrows but the user will still be able to type a negative number in the field. Instead or in addition, we can utilize the validate function of the formik lib to validate: adding validate={value => {... return "errror message"}} to the relevant TextField.

Reply:
Thank you! @galiail
I have made the relevant changes and pushed it. Here is a screenshot that shows the error message when user types a number less than 0.
<img width="517" alt="Screenshot 2023-06-08 at 6 22 12 PM" src="https://github.com/openclarity/kubeclarity/assets/53400750/d235b948-25e9-4bb2-a092-e6523bb519b0">